stat/stat.go

```diff
@@ -97,6 +97,11 @@ func (m Method) String() string {
 	}
 }
 
+// IsMethodValid check range of passed Method variable.
```

Comment:
should be done in the commit that creates that file. i think

Reply:
I'm not sure. The method was added when it was required. In create file commit, this function is unused